index.html

```diff
@@ -53,211 +53,92 @@
-        </article>
-        <article>
-          <h3>Animations</h3>
-          <p>
-            Movement conveys meaning. Through animations we can grab the users
-            attention and make our content more meaningful. We will look at what
-            we can (and should) animate, how CSS animations work and what things
-            we should be paying attention when adding movement to websites.
-            <br />
-            <small
-              >Download slides
-              <a
-                href="./slides/Animations.pdf"
-                target="_blank"
-                rel="noopener noreferrer"
-                >here</a
-              ></small
-            >
-          </p>
-        </article>
-        <article>
-          <h3>Resources, Learning, Getting help</h3>
-          <p>
-            There is too much internet, it’s easy to get lost and not know what
-            to search or learn. This talk aims at showing useful things we have
-            found when scrapping the internet for materials. It will also guide
-            you through how some of us approach learning and asking for help.
-            <br />
-            <small
-              >Download slides
-              <a
-                href="./slides/Resources_Learning_ Getting_Help.pdf"
-                target="_blank"
-                rel="noopener noreferrer"
-                >here</a
-              ></small
-            >
-          </p>
-        </article>
-        <article>
-          <h3>Panel: One year in tech - the newcomers retrospective</h3>
-          <p>
-            So how is working in tech? What are the most challenging things
-            within your first years as a developer? Why not ask some developers
-            who jumped into tech only a few years ago. In this panel, we will
-            reveal all the highs and lows as a junior developer.
-          </p>
-        </article>
-      </section>
-      <section>
-        <h2>Join us</h2>
         <p>
-          This is a truly unique possibility for true development newbies to get
-          in touch with frontend development – join us and become a new dev on
-          the block in no time! Don’t miss out and apply now until June 5 –
-          spaces are limited to 20 participants only!
-          <br />
-          The workshop will be in english and please bring your own notebook.
+          If you wish to attend our workshop, please register <a href="https://forms.gle/kGtsSiLookczkHRq9">here</a>.
         </p>
+        <h3>Venue</h3>
         <p>
-          Click
-          <a href="https://zollhof.typeform.com/to/sxhWgN" target="_blank"
-            >here</a
-          >
-          to apply
+          <address>
+            TBA <br>
+            Vienna
+          </address>
         </p>
-      </section>
-      <section>
+        
+        <section>
+          <h3>Past events</h3>
+          <ol class="past-events">
+            <li><s>Nürnberg, 29th & 30th July 2019</s></li>
```

Comment:
I don't think we need the strikethrough here (it makes it more difficult to read plus it's already clear that it's a past event).

Also: Would we maybe want to create a separate `Resources for NEW DEVS` page with past presentation descriptions and links to the slides? Could be nice for past attendees plus we already have them all available within the repo anyway 🤷‍♀

Reply:
Creating a page with the resources is maybe a good solution.

We could explain, that we changed the format and link to the resources from the first workshop.